manticore/platforms/evm.py

```diff
@@ -2073,14 +2136,14 @@ def transaction(self, address, origin=None, price=0, data='', caller=None, value
             for i in range(len(data)):
                 data_symb[i] = Operators.ORD(data[i])
             data = data_symb
-        bytecode = self.storage[address]['code']
+        bytecode = self.get_code(address)
```

Comment:
not for this PR but change this to `self.account(address).code`?? same with the other `get_` functions?

Reply:
What would self.account(address) return? 
An EVMAccount()?   All the attributes of an EVMAccount may be used as a method name.
So, is .code is the code of the contract or the contract method named 'code' ?